tiup/tiup-mirror.md

```diff
@@ -81,10 +92,117 @@ The `tiup mirror clone` command provides many optional flags (might provide more
     - Execute the `tiup mirror clone <target-dir> --tidb v5.3.0 --tikv all` command to clone the v5.3.0 version of the TiDB component and all versions of the TiKV component.
     - Execute the `tiup mirror clone <target-dir> v5.3.0` command to clone the v5.3.0 version of all components in a cluster.
 
-## Usage examples
+The cloning sets up signing keys as part of the cloning. It is not needed to do this manually.
 
-This section introduces the usage examples of the `mirror` command.
+### Using the private repo
 
 The repository that is cloned using `tiup mirror clone` can be shared among hosts either by sharing the files via SCP, NFS or by making the repository available over the HTTP or HTTPS protocol. Use `tiup mirror set <location>` to specify the location of the repository.
 
-Refer to [Deploy a TiDB Cluster Using TiUP](/production-deployment-using-tiup.md#method-2-deploy-tiup-offline) to install the TiUP offline mirror, deploy a TiDB cluster, and start it.
+```bash
+tiup mirror set /shared_data/tiup
+```
+
+```bash
+tiup mirror set https://tiup-mirror.example.com/
+```
+
+> **Note:**
+>
+> On the machine on which you use `tiup mirror clone` setting the mirror will also change the mirror you are cloning from, so you need to reset this by running `tiup mirror set --reset` before updating the private mirror.
+
+Another way of using a mirror is to use the `TIUP_MIRRORS` environment variable. Here is an example for running `tiup list` with a private repository.
+
+```bash
+export TIUP_MIRRORS=/shared_data/tiup
+tiup list
+```
+
+Using `TIUP_MIRRORS` can permanently change the mirror configuration, just like `tiup mirror set`. For details see [tiup issue #651](https://github.com/pingcap/tiup/issues/651).
+
+### Updating a private repo
+
+When running `tiup mirror clone` a second time with the same `target-dir` it will create new manifests and download newly available versions of components if applicable. It will skip downloading existing component versions.
+
+> **Note:**
+> 
+> As the manifest will be re-created make sure to include all components an all versions (including older ones that were previously downloaded) as otherwise they will be excluded from the manifest.
```

Comment:
```suggestion
> As the manifest will be re-created make sure to include all components and all versions (including older ones that were previously downloaded) as otherwise they will be excluded from the manifest.
```